### ISSUE TITLE
Quantization Aware Training

### ISSUE THREAD

**Issue by ankandrew** [david8862/keras-YOLOv3-model-set#54]
Would it be possible to use Quantization aware training of [TF model optimization](https://www.tensorflow.org/model_optimization/guide/quantization/training) to quantize this implementation of yolo v4?

Looking at [this](https://www.tensorflow.org/lite/performance/model_optimization#quantization) it apparently shows better results in speed-accuracy than Post-training quantization
![Annotation 2020-06-03 015937](https://user-images.githubusercontent.com/61120139/83597579-05acfd80-a53e-11ea-8e8d-8fc658c0246d.png)

**Comment by david8862** [comment]
@ankandrew I've ever tried to check the Quantization aware training support in TF 2.x when it's just published, but at that time it doesn't support many layers & OPs so couldn't be applied directly in this repo. Will take a review to see if it's possible now. Many thanks for suggestion.
